src/frontend/Komponenter/Behandling/Vilkårpanel/GridTabell.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export const GridTabell = styled.div<{
```

Comment:
Vet ikke helt om jeg skjønner forskjellen mellom denne og den andre, hva er målet med en ny?

Reply:
Den er egentlig veldig lik som den eksisterende. Største forskjellen er at det er færre muligheter for å spesialtilpasse gjennom props. F.eks. er det ikke mulig å sette margin under, og den er automatisk 0 fordi all spacing styres fra parent (det er ikke tatt med i denne PR-en). 

Det ble i utgangspunktet laget en kopi med mål om å slette den den gamle etterpå. Slik at det ikke ble noen endringer som ikke fungerte med eksisterende som man ikke la merke til. Siden den gamle brukes så mange ulike steder er det ikke så lett å bare fjerne den. Så kan evt flytte det inn i "TabellVisning" fordi den skal kun brukes der.